src/lib/ai/tools/get-user-info.ts

```
@@ -0,0 +1,63 @@
+import logger from '@/lib/logger';
+import { tool } from 'ai';
```

ESLint

[error] Unable to resolve path to module 'ai'.

(import-x/no-unresolved)